Content.Server/Weapons/Ranged/Systems/GunSystem.cs

```diff
@@ -183,6 +183,11 @@ public override void Shoot(GunComponent gun, List<IShootable> ammo, EntityCoordi
                     {
                         var result = rayCastResults[0];
                         var hitEntity = result.HitEntity;
+
+                        var ev = new HitScanShotEvent(user, hitEntity);
+                        RaiseLocalEvent(ev);
+                        hitEntity = ev.Target;
```

Comment:
How come these aren't directed.

Reply:
Not directed as no EntityUid specified?

Reply:
* option 1 would be use gun entity
* option 2 would be use hit entity but it may change 
so yeah i leave it broadcast